src/containers/BuildUnsignedSweepCoin/BuildUnsignedSweepCoin.tsx

```diff
@@ -141,15 +179,11 @@ function Form() {
             setSubmitting(true);
             try {
               await window.commands.setBitGoEnvironment(bitGoEnvironment);
-              const chainData = await window.queries.getChain(coin);
-              const recoverData = await window.commands.recover(
-                coin,
-                undefined,
-                {
-                  ...(await updateKeysFromIds(coin, values)),
-                  ignoreAddressTypes: ['p2wsh'],
-                }
-              );
+              const chainData = await getChain(coin);
+              const recoverData = await recover(coin, undefined, {
```

Comment:
call window.commands.recover directly, you don't need the try/catch stuff for this call.